syft_tensorflow/hook/hook.py

```diff
@@ -206,6 +213,52 @@ def _hook_keras_model(self, model_cls: type, from_cls: type):
 
         self._hook_keras_methods(model_cls)
 
+    def _hook_variable(self, syft_type: type):
+        """Adds PySyft Tensor Functionality to the given native tensor type.
+         Overloads the given native Torch tensor to add PySyft Tensor
+        Functionality. Overloading involves modifying the tensor type with
+        PySyft's added functionality. You may read about what kind of
+        modifications are made in the methods that this method calls.
+         Args:
+            tensor_type: The type of tensor being hooked (in this refactor
+                this is only ever torch.Tensor, but in previous versions of
+                PySyft this iterated over all tensor types.
+            syft_type: The abstract type whose methods should all be added to
+                the tensor_type class. In practice this is always TorchTensor.
```

Comment:
There are several comments related to Torch in this `_hook_variable` method that we should remove.